README.md

```diff
@@ -1,17 +1,25 @@
-# GAN Metrics
+GAN Metrics
+------

```

Comment:
I prefer to keep the title a large heading.

Reply:
You have the conn.